data/presets/presets/amenity/parking/underground.json

```diff
@@ -18,11 +18,6 @@
         "parking": "underground",
         "layer": "-1"
     },
-    "removeTags": {
-        "amenity": "parking",
-        "parking": "underground",
-        "layer": "-1"
-    },
```

Comment:
Would make `"layer": "*"` make sense here for `"removeTags"`? Then it should stay.

Reply:
Actually in `removeTags` the values don't mean anything, only the keys.

Reply:
Oh, then I can remove some more `"removeTags"`, except if you think they should be kept for better overview?

Reply:
Well, I think I will leave it because maybe in the future iD wants to only remove a tag if the value hasn't been changed to something else in the meantime. Removing it feels like a loss of information of the preset.